build/scripts/wait.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+gcloud container clusters get-credentials ${CLUSTER_NAME}
+
+echo "Waiting for the load balancer to be accessible (expected time: ~1min)"
+DEPLOYED_APP_IP=""
+while [[ -z "$DEPLOYED_APP_IP" ]]; do
+  sleep 5
```

Comment:
It might be good to print a progress message inside the loop as well to let us know that something is happening.